src/components/map/MapComponent.vue

```diff
@@ -178,6 +170,13 @@ viewStore.$subscribe(async () => {
 mapStore.$subscribe(async () => {
   if (rennesApp.maps.activeMap.name !== mapStore.activeMap) {
     await rennesApp.maps.setActiveMap(mapStore.activeMap)
+
+    if (
```

Comment:
Thanks, @toni-garcia-camptocamp 
I understand the problem now. It's because we activate the 3D map via mapStore. We already set the mapStore to 3D, but the process to activate the 3D map itself is not finished because we do it via a store subscription.

IMHO, we should make sure the switch to 3D when we want to compute the height so that it has a more logical flow. If we put it here, it's random to put the solar panel fixtures setup but the zoom is on the other location.

Anyway, really thanks for the fix! I will merge this.